Refrigerator.cs

```diff
@@ -0,0 +1,279 @@
+            throw new Exception("Item is not exist!");
+
+        }
+
+        public List<Item> GetItemsToEat(string kashruth, string type)
+        {
+            this.CleanRefrigerator();
+            List<Item> itemsToEat = new List<Item>();
+            foreach (Shelf shelf in shelves)
+            {
+                var items = shelf.GetItemsToEat(kashruth, type);
+                if (items != null)
+                {
+                    itemsToEat.AddRange(items);
+                }
+            }
+            return itemsToEat;
+        }
+        public bool WhatShouldIEat(string kashruth, string type)
+        {
+            var matchFood = GetItemsToEat(kashruth, type);
+            if (matchFood.Count != 0)
+            {
+                foreach (Item item in matchFood)
+                {
+                    Console.WriteLine(item);
+                }
+            }
+            else
+            {
+                Console.WriteLine("There are no products matching your request.");
+                return false;
+
+            }
+            return true;
+        }
+        public double GetTotalSpace(List<Item> items)
+        {
+            double sumSpace = items.Sum(item => item.space);
+
+            return sumSpace;
+
+        }
+
+        public List<Item> DateAndTypeClean(DateTime date, string type)
+        {
+            List<Item> items = new List<Item>();
+            foreach (Shelf shelf in shelves)
+            {
+                var subList = shelf.items.FindAll(item => item.expirationDate < date && item.kashruth == type);
+                if (subList != null)
+                {
+                    items.AddRange(subList);
+                }
+            }
+            return items;
+        }
+        public List<Item> CleanDairy()
+        {
+            var result = this.DateAndTypeClean(DateTime.Today.AddDays(+3), "Dairy");
+
+            return result;
+        }
+
+        public List<Item> CleanMeat()
+        {
+            var result = this.DateAndTypeClean(DateTime.Today.AddDays(+7), "Meat");
+
+            return result;
+        }
+        public List<Item> CleanPareve()
+        {
+            var result = this.DateAndTypeClean(DateTime.Today.AddDays(+1), "Pareve");
+
+            return result;
+        }
+        public bool ItsEnough(List<Item> itemToRemove)
+        {
+
+            if (this.GetTotalSpace(itemToRemove) >= 20)
+            {
+                return true;
+            }
+            return false;
+        }
+        public void printAndRemove(List<Item> itemToRemove)
+        {
+            foreach (Item item in itemToRemove)
+            {
+                Console.WriteLine(item.ToString());
+                RemoveFromRefrigerator(item.id);
+
+            }
+        }
+        public void PrintCleanStatus(List<Item> items)
+        {
+            if (items.Count > 0)
+            {
+                Console.WriteLine("As part of cleaning, we throw those items that had not expired:");
```

Comment:
delete the word "not"

Reply:
why? These are supposed to be fresh products... 
How about: As part of the cleaning process, we discard items that are unexpired

Reply:
Refrigerator.CleanRefrigerator () collects items by Shelf.CleanShelf that return expired items, 
Then it calls PrintCleanStatus with these items, so it prints expired items.
If the names of functions were better we won't have this confusion.